src/quantize.rs

```diff
@@ -83,7 +83,7 @@ pub fn select_dc_qi(quantizer: i64, bit_depth: usize) -> u8 {
     8 => &dc_qlookup_Q3,
     10 => &dc_qlookup_10_Q3,
     12 => &dc_qlookup_12_Q3,
-    _ => unimplemented!(),
+    _ => return 255,
```

Comment:
This could be handled in rate.rs, selecting a quantizer index for a non-existent bitdepth is strange.

Reply:
That's just for testing.
The real fix would be modifying the rate control system. Rust makes this so not nice.